week-5-practise/js/main.js

```diff
@@ -0,0 +1,66 @@
+var btnBlue = document.querySelector("#blueBtn");
+var btnOrange = document.querySelector("#orangeBtn");
+var btnGreen = document.querySelector("#greenBtn");
+
+var bgJumbotron = document.querySelector(".primary-content .jumbotron");
+var bikeDonate = document.querySelector(".primary-content .btn-primary");
+var volunteerButton = document.querySelector(".primary-content .btn-secondary");
+
+btnBlue.addEventListener("click", function() {
+  bgJumbotron.style.backgroundColor = "#588fbd";
+  bikeDonate.style.backgroundColor = "#ffa500";
+  volunteerButton.style.backgroundColor = "black";
+  volunteerButton.style.color = "white";
+});
+
+btnOrange.addEventListener("click", function() {
+  bgJumbotron.style.backgroundColor = "#f0ad4e";
+  bikeDonate.style.backgroundColor = "#5751fd";
+  volunteerButton.style.backgroundColor = "#31b0d5";
+  volunteerButton.style.color = "white";
+});
+
+btnGreen.addEventListener("click", function() {
+  bgJumbotron.style.backgroundColor = "#87ca8a";
+  bikeDonate.style.backgroundColor = "black";
+  volunteerButton.style.backgroundColor = "#8c9c08";
+});
+
+var btnSubmit = document.querySelector("form button");
+var formName = document.querySelector("#example-text-input");
+var formComment = document.querySelector("#exampleTextarea");
+var formMail = document.querySelector("#exampleInputEmail1");
+
+var allFields = document.querySelectorAll("form .form-control");
+
+btnSubmit.addEventListener("click", function() {
+  event.preventDefault();
```

Comment:
I don't think `event` is defined here. But I actually think the default behaviour (which you're trying to prevent) is what we want. Or at least it is if you're listening for click events on the button.